tests/e2e/specs/settings/applications-passwords.test.js

```diff
@@ -0,0 +1,110 @@
+import {
+    visitAdminPage,
+    __experimentalRest as rest,
+} from '@wordpress/e2e-test-utils';
+
+async function getResponseForApplicationPassword() {
+    return await rest({
+        method: 'GET',
+        path: '/wp/v2/users/me/application-passwords'
+    });
+}
+
+async function createApplicationPassword(applicationName) {
+    await visitAdminPage('profile.php');
+    await page.waitForSelector('#new_application_password_name');
+    await page.type('#new_application_password_name', applicationName);
+    await page.click('#do_new_application_password');
+
+    await page.waitForSelector('#application-passwords-section .notice');
+}
+
+async function revokeAllApplicationPasswords() {
+    await visitAdminPage('profile.php');
+
+    const revokeAllButtonVisibility = await page.evaluate(() => {
+        const e = document.querySelector('.application-passwords-list-table-wrapper');
+        if (!e) {
+            return false;
+        }
+        const visibility = window.getComputedStyle(e);
+        return visibility && visibility.display !== 'none' && visibility.visibility !== 'hidden' && visibility.opacity !== '0';
+    });
+
+    if (revokeAllButtonVisibility) {
+        await page.click('#revoke-all-application-passwords');
+        await page.keyboard.press('Enter');
+        await page.waitForSelector('#application-passwords-section .notice-success');
+    }
+}
+
+async function revokeAllApplicationPasswordsWithApi() {
+    const response = await getResponseForApplicationPassword();
+    const uuids = response.map(applicationPassword => applicationPassword.uuid);
+    for (const uuid of uuids) {
+        await rest({
+            method: 'DELETE',
+            path: `/wp/v2/users/me/application-passwords/${uuid}`
+        });
+    }
+}
+
+describe('Manage applications passwords', () => {
+    const testApplicationName = 'Test Application';
```

Comment:
Nit: Use ALL CAPS for constant variable name.

```suggestion
    const TEST_APPLICATION_NAME = 'Test Application';
```